index.html

```diff
@@ -224,37 +224,26 @@ <h2>Multi-Post Stories</h2>
     </div>
   </section>
   <section class="contact">
-    <div class="contact-us">
-      <h2>CONTACT US</h2>
-    </div>
-    <!-- paragraph 3 -->
-    <div class="paragraph3">
-      <p>If you have an application you are <br>
+    <article>
+      <h2 class="contactus">CONTACT US</h2>
+      <p class="paragraph3">If you have an application you are <br>
         interested in developing, a feature that you <br>
         need built or a project that needs coding. <br>
         I’d love to help with it </p>
-    </div>
-    <ul class="buttom">
-      <li class="input-text">
-        input-text
-      </li>
-    </ul>
-
-    <ul class="buttom">
-      <li class="input-email">
-        jeremy@gmail.com
-      </li>
-    </ul>
-    <ul class="buttom">
-      <li class="input-comment">
-        input text here
-      </li>
-    </ul>
-    <button class="btn2">See project</button>
-    <!-- line4 -->
-    <hr>
-<!-- social media2 -->
-<div class="section-one-column2">
+    </article>
+    <form>
+      <ul>
+        <li><input type="text"></li>
+        <li><input type="email"></li>
+        <li><textarea type="text" name="" id="" cols="30" rows="10"></textarea></li>
```

Comment:
- [ ] Please add type and placeholder attributes to your input fields also you need to tag them as required and give some validations for each as requested in the project requriements: 
![image](https://user-images.githubusercontent.com/91486791/194123208-bd4a3c66-0650-41d5-a947-5d83b716eae6.png)

- This would increase the user experience and validate all inputs are filled with the right formats.